### PR TITLE
[MBO-152] Correctly translate the addons link for carriers & payment

### PR DESCRIPTION
**Pull request by intraordinaire**
TO have the correct link in all languages.
Maybe this is a correct solution to handle translated links ?
What do you think @sowbiba  ?

### CODE REVIEW

File: src/Controller/Admin/ModuleRecommendedController.php [hunk 1]
```diff
@@ -93,4 +101,27 @@ public function indexAction(): JsonResponse
...
+     *
+     * @return string|null
+     */
+    protected function getAddonsLink(string $tabClassName): ?string
+    {
+        $utm = '?utm_source=back-office&utm_medium=dispatch&utm_content=download&utm_campaign=back-office-' . $this->context->getLanguage()->iso_code;
+
+        switch ($tabClassName) {
+            case 'AdminCarriers':
+                $link = $this->trans('https://addons.prestashop.com/en/518-shipping-logistics', 'Modules.Mbo.Modulescatalog');
```
Comment (sowbiba) [inline src/Controller/Admin/ModuleRecommendedController.php:116]: there an `/en/` here to replace with the iso_code

**Comment by sowbiba** [comment]
> TO have the correct link in all languages. Maybe this is a correct solution to handle translated links ? What do you think @sowbiba ?

Yes, this is a better solution than generating the link in the template

You have https://github.com/PrestaShopCorp/ps_mbo/blob/3.x/src/Addons/Provider/LinksProvider.php#L31 
You can maybe implement `getAddonsLink` here ? wdyt ?

File: src/Controller/Admin/ModuleRecommendedController.php (hunk 1 above)
Comment (intraordinaire) [reply src/Controller/Admin/ModuleRecommendedController.php:116]: In french, the page is `/fr/518-livraison-logistique`
In Italian : `/it/518-spedizioni-logistica`
And so on...
That's why I put the link in the translation module.
Each language may update the URL of those pages.

**Comment by intraordinaire** [comment]
> > TO have the correct link in all languages. Maybe this is a correct solution to handle translated links ? What do you think @sowbiba ?
> 
> Yes, this is a better solution than generating the link in the template
> 
> You have https://github.com/PrestaShopCorp/ps_mbo/blob/3.x/src/Addons/Provider/LinksProvider.php#L31 You can maybe implement `getAddonsLink` here ? wdyt ?

Will do it

**Review by sowbiba — APPROVED** [review]

**Pull request merged**